@here/harp-mapview/lib/DecodedTileHelpers.ts

```diff
@@ -116,6 +121,7 @@ export function createMaterial(
     ) {
         settings.fog = options.fog;
     }
+    settings.removeDiffuseLight = technique.name === "fill";
```

Comment:
While extra param usually doesn't harm, i think it should be guarded with If `options.shadowsEnabled`.

Reply:
Agreed.